projects/fourier/lib/fourier/services/bundle/tuist.rb

```diff
@@ -41,6 +41,8 @@ def call
                 xcode_paths: xcode_paths,
               )
 
+              # TODO: build xcbeautify
```

Comment:
@pepicrft as you are refactoring the release process I think it make sense to do this after that is ready

Reply:
Sure! I added a note there so I don't forget about it. Thanks for bringing it up.